src/test/java/chapter7/question1/Question1Test.java

```diff
@@ -0,0 +1,27 @@
+package chapter7.question1;
+
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.*;
+import src.chapter7.question1.Question1;
+
+public class Question1Test {
+    @Test
+    public void isEven_whenANumberIsEven_shouldReturnTrue() {
+        assertEquals(true, Question1.isEven(10));
```

Comment:
You can replace this for assertTrue(Question1.isEven(10));

Reply:
FIxed.